src/Components/BackToScreen/BackToScreen.css

```diff
@@ -12,3 +12,10 @@
   font-size: 1rem;
   cursor: pointer;
 }
+
+.back-to-screen-warning-icon {
+  min-width: 2rem;
+  min-height: 2rem;
```

Comment:
What do you think about removing these properties since the width and height are hard coded to be 2rems below?

Reply:
I tried that, but it did not work.